app/rubin-2025/register-form.jsx

```diff
@@ -79,233 +79,294 @@ class RegisterForm extends React.Component {
-              <span className="form-help error">
-                <Translate content="registerForm.emailConflict" />{' '}
-                <a href={`${window.location.origin}/reset-password`}>
-                  <Translate content="registerForm.forgotPassword" />
-                </a>
-              </span>
-            )}
+
+            <div aria-live="polite" id="register-form-email-help-message-3">
+              {emailConflict && (
+                <span className="form-help error">
+                  <Translate content="registerForm.emailConflict" />{' '}
+                  <a href={`${window.location.origin}/reset-password`}>
+                    <Translate content="registerForm.forgotPassword" />
+                  </a>
+                </span>
+              )}
+            </div>
+
             {(this.state.input_email?.length > 0
               && !emailConflict
               && !emailInvalidChars
               && !emailInvalidFormat
+              && !('emailConflict' in this.state.pending)
             ) && (
               <Translate className="form-help success" content="registerForm.looksGood" />
             )}
+
             <Translate className="form-help info right-align" content="registerForm.required" />
-          </span>
+          </div>
+
           <input
-            type="text"
-            name="email"
-            id="register-form-email"
-            value={this.state.input_email}
+            aria-describedby="register-form-error-message register-form-email-help-message-1 register-form-email-help-message-2 register-form-email-help-message-3"
             className="standard-input full"
             disabled={inputDisabled}
+            id="register-form-email"
+            name="email"
             onChange={this.handleUserInput}
+            type="text"
+            value={this.state.input_email}
           />
-        </label>
-
-        <br />
+        </div>
 
-        <label>
-          <span className="columns-container inline spread">
-            <Translate content="registerForm.realName" />
+        <div className="form-row">
+          <div className="label-block">
+            <label htmlFor="register-form-creditedName">
+              <Translate content="registerForm.realName" />
+            </label>
             <Translate className="form-help info right-align" content="registerForm.optional" />
-          </span>
+          </div>
           <input
-            type="text"
-            pattern='[^@]+'
-            name="creditedName"
-            id="register-form-creditedName"
-            value={this.state.input_creditedName}
             className="standard-input full"
             disabled={inputDisabled}
-            title={counterpart('registerForm.realNamePatternHelp')}
+            id="register-form-creditedName"
+            name="creditedName"
             onChange={this.handleUserInput}
+            pattern='[^@]+'
+            title={counterpart('registerForm.realNamePatternHelp')}
```

Comment:
>If you want to hide content from mobile and tablet users as well as assistive tech users and keyboard only users, use the title attribute.

https://www.tpgi.com/using-the-html-title-attribute-updated/

This title text is hidden from phones, tablets, and screen readers.

Reply:
![Obi Wan Kenobi: Title? That's an attribute I haven't seen in a long time.](https://github.com/user-attachments/assets/f3cacb3f-b000-4ca1-9643-787c77a6fcbe)